test/athens/events/awaited_events_test.cljs

```diff
@@ -0,0 +1,146 @@
+#_{:clj-kondo/ignore [:unused-namespace]}
+
+
+(ns athens.events.awaited-events-test
+  "Testing `awaited-event` events and subscriptions."
+  (:require
+    [athens.effects]
+    [athens.events]
+    [athens.events.fixture :as fixture]
+    [athens.subs]
+    [cljs.test             :refer-macros [deftest is]]
+    [day8.re-frame.test    :as rf-test]
+    [re-frame.core         :as rf]))
+
+
+(deftest initial-state-not-awaiting-any-events
+  (rf-test/run-test-sync
+    (fixture/test-fixtures)
+    (rf/dispatch [:boot/web])
+    (let [awaited-events (rf/subscribe [:remote/awaited-events])]
+      (is (not (nil? @awaited-events)))
+      (is (empty? @awaited-events)))))
+
+
+(deftest awaiting-one-event
+  (rf-test/run-test-async
+    (fixture/test-fixtures)
+    (rf/dispatch-sync [:boot/web])
+    (let [awaited-events (rf/subscribe [:remote/awaited-events])]
```

Comment:
might be useful to share this setup boilerplate to make each test a bit shorter.

Reply:
also might be useful to factor into one function the test of "is this an empty collection" (e.g., like on lines 32,33), or the test of "is a collection containing this single element"